db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -268,10 +268,10 @@ Status DBImpl::FlushMemTableToOutputFile(
           error_handler_.SetBGError(io_s,
                                     BackgroundErrorReason::kManifestWriteNoWAL);
```

Comment:
Should the same change be done here, i.e ignore ```total_log_size_``` and always set reason to ```kManifestWriteNoWAL```?

Reply:
> Should the same change be done here, i.e ignore `total_log_size_` and always set reason to `kManifestWriteNoWAL`?

Yes